types/src/traits/election.rs

```diff
@@ -53,6 +53,7 @@ pub enum Checked<T> {
 
 /// Data to vote on for different types of votes.
 #[derive(Serialize)]
+// TODO (da) : LEAF type should just be changed to a commitment and bounded by Commitable
```

Comment:
Yeah you are right it doesn't make sense.  I was going through some stuff that would have to change for having 2 proposal types and for some reason got to this and thought it might need to change, but looking at it now I realize it does not.